tests/l2/README.md

```diff
@@ -113,10 +113,12 @@ $ oc logs intel-dgpu-hwinfo-l4572
 ```                        
 
 ### Verify Intel® QuickAssist Technology provisioning
-This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
+This workload runs [qatlib](https://github.com/intel/qatlib) sample tests using RPM packages. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
 
 *	Build the workload container image
 
+Please replace the credentials in buildconfig yaml with your RedHat account login credentials. Updates with entitlement builds and enhancements for credentials as secret/build inputs will be added in the next releases.
```

Comment:
Can you add the link about entitlement builds and enhancements for credential as secret/build? Or
Do you think we can just remove the second sentence about entitlement builds and enhance ments?

Reply:
Sure, we can remove it. entitlement builds has not worked as of yet. enhancement to pass credentials as secret is possible.

Reply:
updated